tests/ui/pages/desktop/base.py

```diff
@@ -90,6 +100,20 @@ def click_logout(self):
         self.wait.until(lambda s: self.is_element_displayed(
             *self._login_locator))
 
+    def more_menu(self, item=None):
+        menu = self.find_element(*self._more_dropdown_locator)
+        links = menu.find_elements(*self._more_dropdown_link_locator)
+        action = ActionChains(self.selenium)
+        action.move_to_element(menu)
+        action.pause(2)
+        action.click()
+        action.pause(2)
+        action.move_to_element(links[item])
+        action.pause(2)
+        action.click()
+        action.pause(2)
+        action.perform()
```

Comment:
Can we add a few comments for the separate steps what which step does? Simply reading this method makes it really hard to follow along.